packages/testlab/src/__tests__/unit/to-json.test.ts

```diff
@@ -62,6 +62,50 @@ describe('toJSON', () => {
     expectUndefined(value);
   });
 
+  it('handles `object | null`', () => {
+    const input: object | null = Math.random() ? {} : null;
+    const output = toJSON(input);
+    expectComplexType<object | null>(output, input);
+  });
+
+  it('handles `object | undefined`', () => {
+    const input: object | undefined = Math.random() ? {} : undefined;
```

Comment:
Please extract `Math.random()` ? to a helper function, such as: `randomChoice(...values: any[]): any`

Reply:
For example:
```ts
function randomChoice(...values: any[]): any {
  if (!values.length) return undefined;
  const index = Math.floor(Math.random() * values.length);
  return values[index];
}
```